src/subscription_manager/migrate/migrate.py

```diff
@@ -642,7 +642,7 @@ def legacy_unentitle(self, rpc_session):
     def legacy_purge(self, rpc_session, session_key):
         system_id_path = self.rhncfg["systemIdPath"]
 
-        log.info("Deleting system %s from legacy server...", self.system_id)
+        log.debug("Deleting system %s from legacy server...", self.system_id)
```

Comment:
Deleting system from legacy server is also change of state. Please keep `info` here too. Also consider similar changes in this file. Migration from RHN to RHSM isn't anything costumers do everyday and they probably want to have as much information about it as possible by default. Thoughts?